examples/export/export-image-gdrive.ipynb

```diff
@@ -2,13 +2,13 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 7,
```

Comment:
The cells appear to have been run out of order.
If you are saving a notebook file with outputs, be sure to reset the kernel, clear the outputs, and run all the cells before saving and committing the file.